src/test/java/com/answerdigital/benhession/academy/answerkingweek2/controllers/OrderControllerTest.java

```diff
@@ -0,0 +1,109 @@
+package com.answerdigital.benhession.academy.answerkingweek2.controllers;
+
+import com.answerdigital.benhession.academy.answerkingweek2.request.AddOrderRequest;
+import com.answerdigital.benhession.academy.answerkingweek2.services.OrderService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+@WebMvcTest(OrderController.class)
+public class OrderControllerTest {
+    @Autowired
+    private OrderController orderController;
+
+    @MockBean
+    private OrderService orderService;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    /**
+     * Method under test: {@link OrderController#addItemToBasket(Long, Long, Integer)}
+     */
+    @Test
+    void testAddItemToBasket() throws Exception {
```

Comment:
I think this method can be shortened down to:
@Test
    @WithMockUser("paul")
    void testAddItemToBasket() throws Exception {
        
        mockMvc.perform(post("/{orderId}/item/{itemId}/quantity/{quantity}", 123L, 123L, 1))
                .andExpect(MockMvcResultMatchers.status().isNotFound());
    }

It is shorter and more readable and hence it is easier to maintain it in the future. Similarly all the other method in this class.